Content.Client/Cargo/UI/CargoBountyMenu.xaml.cs

```diff
@@ -32,5 +36,19 @@ public void UpdateEntries(List<CargoBountyData> bounties, TimeSpan untilNextSkip
         {
             MinHeight = 10
         });
+
+        BountyHistoryContainer.Children.Clear();
+        if (history.Count == 0)
+        {
+            NoHistoryLabel.Visible = true;
+        }
+        else
+        {
+            NoHistoryLabel.Visible = false;
+            foreach (var h in history)
+            {
+                BountyHistoryContainer.AddChild(new BountyHistoryEntry(h));
+            }
+        }
```

Comment:
What are your thoughts on reversing the order that history is displayed? 

I think it would make more sense for newer history to be at the top rather than the bottom.

Reply:
Sounds like a good idea 🤔 !